website/docs/deployment/traefik.md

```
@@ -0,0 +1,210 @@
+  }'
+```
+
+### Check Traefik Logs
+
+```bash
+docker logs traefik -f
+```
+
+Look for lines like:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)